lib/internal/process/esm_loader.js

```diff
@@ -30,6 +30,7 @@ exports.importModuleDynamicallyCallback = async function(wrap, specifier) {
       'ExperimentalWarning', undefined);
     calledInitialize = true;
```

Comment:
seems like this line can be removed, since line 33 exists?
```suggestion
```

Reply:
Thanks